infra/gp-grafana-instance/values.yaml

```diff
@@ -0,0 +1,14 @@
+# Default values for gp-grafana-instance.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+grafana:
+  auth:
+    admin:
+      user: "admin"
+      password: "insert some password here"
```

Comment:
Muss man hier das Passwort setzen?
Falls nicht wird eines generiert? Wenn ja, wo kann man es auslesen?

Reply:
Nein, ja. In https://gepardec.atlassian.net/wiki/spaces/G/pages/2460352513/Grafana#Authentifizierung beschrieben